oeps/oep-0006.rst

```diff
@@ -0,0 +1,224 @@
+===================================
+OEP-6: Context-scoped XBlock Fields
+===================================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`OEP-6 </oeps/oep-0006>`             |
++---------------+-------------------------------------------+
+| Title         | Context-scoped XBlock Fields              |
++---------------+-------------------------------------------+
+| Last Modified | 2016-08-03                                |
++---------------+-------------------------------------------+
+| Author        | Braden MacDonald <braden@opencraft.com>   |
++---------------+-------------------------------------------+
+| Arbiter       |                                           |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Architecture                              |
++---------------+-------------------------------------------+
+|  Created      | 2016-08-03                                |
++---------------+-------------------------------------------+
+| `Resolution`  |                                           |
++---------------+-------------------------------------------+
+
+Abstract
+========
+
+There is currently no standard way for XBlocks to define data fields such that content, settings, or user input can be shared among multiple XBlocks within a particular "context" such as a course. Such "course-scoped" or "context-scoped" fields are a common requirement; for example, an author may wish to specify the default “number of attempts allowed” once for all problems in a course, or a video player XBlock may want to “remember” the user’s preferred video playback speed for videos within a particular course.
```

Comment:
I tend to prefer wrapped lines in the raw text, simply because it's easier to read in an editor (and in the email summaries that github sends). Github wraps them nicely in the diff, but in this case I lean towards optimizing towards the lowest common denominator.